cirq/ops/projectors_test.py

```diff
@@ -0,0 +1,114 @@
+import re
+
+import numpy as np
+import pytest
+
+import cirq
+
+
+def test_projector_circuit_diagram():
+    q = cirq.NamedQubit('q')
+    projector = cirq.Projector([[1.0, 0.0]])
+    cirq.testing.assert_has_diagram(cirq.Circuit(projector(q)),
```

Comment:
How do two-qubit projectors look like?

Reply:
Added a test for 2 qubits, code was crashing, fixed the code.